packages/quantic/cypress/page-objects/search.ts

```diff
@@ -90,6 +94,18 @@ export const InterceptAliases = {
     UndoQuery: uaAlias('undoQuery'),
     SearchboxSubmit: uaAlias('searchboxSubmit'),
   },
+  NextAnalytics: {
+    Qna: {
+      AnswerAction: nextAnalyticsAlias('Qna.AnswerAction'),
```

Comment:
does AnswerAction encompasses everything else for Qna? like RetryGeneratedAnswer, ShowGeneratedAnswer, HideGeneratedAnswer, etc. ?